src/components/App/App.js

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 
 import ToastPlayground from '../ToastPlayground';
 import Footer from '../Footer';
+import ToastProvider from '../ToastProvider';
 
 function App() {
-  return (
-    <>
-      <ToastPlayground />
-      <Footer />
-    </>
-  );
+	return (
```

Comment:
I cannot fathom why it shows up like this in the diff, as my indentation looks the same locally in VSC 🤷‍♀️